docs/local-tableland/fork-chain.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Forking chain state
+description: Fork a live testnet or mainnet chain and use its state on Local Tableland.
+keywords:
+  - fork
+  - fork chain
+  - local tableland
+---
```

Comment:
This is the primary content—walks through the forking params, example test, and calls out that testnets like Mumbai should be avoided.